docs/UserGuide.md

```diff
@@ -252,15 +260,19 @@ Sample Output:
 ---
 ### Searching for an applicant : `search`
 
-Allows you to search for applicants containing specified keywords, these keywords could be in **any** of the 
-applicant's name, job, round or skills.
+Allows you to search for applicants containing the input keywords and the keyword has to be an exact word. You can search for the applicant based on 
```

Comment:
@zzhikai I think in our GUI pic at the start, would be good to label what is a keyword so that people who read aren't confused. Not sure if using keywords vs input will cause confusion.